src/cpp/client_spawner/WorkerClientNode.cc

```diff
@@ -0,0 +1,65 @@
+#include "WorkerClientNode.h"
+#include "SentinelServerNode.h"
+#include "DASNode.h"
+#include "RemoteIterator.h"
+#include "QueryAnswer.h"
+
+using namespace client_spawner;
+using namespace query_engine;
+using namespace query_element;
+
+#define MAX_QUERY_ANSWERS ((unsigned int) 1000)
+
+WorkerClientNode::WorkerClientNode(
+    const string &node_id,
+    const string &server_id,
+    const string &das_node_server_id
+) : StarNode(node_id, server_id) {
+        this->node_id = node_id;
+        this->server_id = server_id;
+        this->das_node_server_id = das_node_server_id;
+}
```

Comment:
`StarNode` is already storing `server_id` (attr has the same name), and `DistributedAlgorithmNode` (parent of `StarNode`) is also storing `node_id` as `my_node_id` (private) but accessible via `node_id()` function.
```suggestion
WorkerClientNode::WorkerClientNode(
    const string &node_id, const string &server_id, const string &das_node_server_id
) : das_node_server_id(das_node_server_id), StarNode(node_id, server_id) {}
```